examples/complete/variables.tf

```diff
@@ -1,4 +1,38 @@
-variable "example" {
+variable "branch" {
+  description = "A reference to your VCS repository in the format `:org/:repo` where `:org` and `:repo` refer to the organization and repository in your VCS provider."
+  type        = string
+}
+
+variable "identifier" {
```

Comment:
are `branch` and `identifier` have the correct descriptions?

Reply:
also, if "Default to `master`", then maybe we need to provide the default, otherwise TF will ask for the var

Reply:
Fixed!